src/main/java/com/salesforce/storm/spout/dynamic/kafka/Consumer.java

```diff
@@ -256,19 +256,12 @@ public void open(
             } else {
                 // We do not have an existing offset saved, so start from the head
                 getKafkaConsumer().seekToBeginning(Collections.singletonList(topicPartition));
-                offset = getKafkaConsumer().position(topicPartition) - 1;
 
                 logger.info(
                     "Starting at the beginning of namespace {} partition {} => offset {}",
                     topicPartition.topic(), topicPartition.partition(), offset
                 );
             }
-
-            // Start tracking offsets on ConsumerPartition
```

Comment:
Why drop this block?

Reply:
With position() failing there would be nothing to store, but that's a fair concern - maybe we can be more intuitive and check if we actually set offset to anything first.